interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -123,7 +123,7 @@
 <body bgcolor="{$STYLE.BGCOLOR2}">
 <p><table><tr><td><span class="title">{xl t="Vitals"}</span></td><td>&nbsp;&nbsp;&nbsp;<a href="../summary/demographics.php" class="readonly css_button_small" onclick="top.restoreSession()"> <span>{xl t="View Patient"}</span></a></td></tr></table></p>
 <form name="vitals" method="post" action="{$FORM_ACTION}/interface/forms/vitals/save.php" onSubmit="return vitalsFormSubmitted()">
-<div id="chart"></div>
+<div id="chart" style="direction:ltr;"></div>
```

Comment:
can't do this for now since this will go into a 5.0.0 patch.
But after initial commit will then do this for 5.0.1 (will create a class graph_dygraphs and then use this for all the graphing elements and place the ltr here in core.css)